desktop/cmp/panel/PanelModel.js

```diff
@@ -111,16 +111,11 @@ export class PanelModel {
 
         apiRemoved(rest.prefName, 'prefName', 'Specify "persistWith" instead.');
 
-        if (!isNil(maxSize) && (maxSize < minSize || maxSize < defaultSize)) {
```

Comment:
This doesn't appear to be necessary anymore.   The panel will just respect the min/max style settings that are put onto the cmp in ResizeContainer.js, and not be resizable if the defaultSize does not fall with them.